Documentation/users/saml-integration.md

```diff
@@ -0,0 +1,133 @@
+
+1. Use kubectl to back up the existing config:
+
+```
+kubectl get configmaps tectonic-identity --namespace=tectonic-system -o yaml > tectonic-config.yaml.bak
+```
+
+2. Edit the current `ConfigMap` with the desired changes:
+
+```
+kubectl edit configmaps tectonic-identity --namespace=tectonic-system
+```
+
+3. Add the `connectors` configuration.
+
+Replace `example.com` with your IdP domain name, and `tectonic-domain` with your Tectonic domain name.
+
+```yaml
+connectors:
+- type: saml
+  id: saml
+  name: SAML
+  config:
+    ssoURL: https://example.com/sso/saml
+    redirectURI: https://tectonic-domain/identity/callback
+    usernameAttr: name
+    emailAttr: email
+    groupsAttr: groups # optional
+    caData: /path/to/ca.pem
+    entityIssuer: https://tectonic-domain/identity/callback
+    ssoIssuer: https://example.com/sso/saml
+    nameIDPolicyFormat: persistent
+```
+
+Run `kubectl apply` to apply the new configuration:
+
+```
+kubectl apply -f path/to/config/file/new-tectonic-config.yaml
+```
+
+Finally, trigger a rolling update of the Identity pods, which will read the new configuration:
+
+```
+kubectl patch deployment tectonic-identity \
+    --patch "{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"date\":\"`date +'%s'`\"}}}}}" \
+    --namespace tectonic-system
+```
+
+If successful the following message is displayed:
+
+```
+"tectonic-identity" patched
+```
+
+Create role bindings to allow SAML users access to Kubernetes resources through both Tectonic Console and kubectl. See [Configuring RBAC](#configuring-rbac) for more details.
+
+### Using kubectl as a SAML user
+
+To use kubectl as a SAML user, log in to Tectonic Console as the desired user. Then follow the standard process to [download kubectl and configure credentials][kubeconfig-download] for the cluster.
+
+### Handling SAML authentication expiration
+
+Every twenty four hours, access tokens for the session will expire. Users must log in again to regain access to the cluster.
+
+## Tectonic Identity configuration parameters
+
+Parameters must be configured on both Tectonic Identity and the SAML IdP to enable them to exchange user data. These parameters are defined for Tectonic Identity in the `config-dev.yaml` file, which must be mapped directly to those configured on the IdP side.
+
+Once configured, SAML IdP exchanges security tokens with Tectonic Identity, acting as a SAML consumer  to exchange user authentication data.
+
+See your provider's IdP documentation for more information on their specific SAML configuration options.
+
+The table below describes the parameters that configure Tectonic Identity federation with IdPs:
+
+question: are these platform agnostic? in which case, i want to pull them from here, and add them to tectonic-identity-config.md - beth
```

Comment:
These are platform agnostic but really cryptic. Let me take a second pass at the SAML section in another review.